src/routes/index.js

```diff
@@ -29,6 +29,8 @@ import {
   updateUserPassword,
 } from './users';
 import swaggerDocument from '../swagger.json';
+import verifyToken from '../services/verifyToken.js';
```

Comment:
Can we rename the services folder to middleware or something more descriptive? Im worried that having a services and utils folder can be confusing. And also can you add a description of the new folder to project structure part of the readme?